### DISCUSSION TITLE
Differing documentation on changelogUrl

### DISCUSSION THREAD

**Discussion by jvmdc** [renovatebot/renovate#30299] [Request Help]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Bitbucket Server - Renovate 37.431.4

### Please tell us more about your question or problem

The documentation on the "Key concepts" -> "Renovate and changelogs" page (https://docs.renovatebot.com/key-concepts/changelogs/#changelogurl) states the following about `changelogUrl`: 
> This doesn't help with fetching the changelogs, but if you configure it then Renovate will include a link to this URL in the PR body, so users can click through to read the changelog.

The page "Configuration" -> "Repository" (https://docs.renovatebot.com/configuration-options/#changelogurl) says the following on `packageRules` -> `changelogUrl`: 

> If set, Renovate will use this URL to fetch changelogs for a matched dependency. Valid only within a packageRules object.
> [...]
> Use this field to set the changelog URL for a package, including overriding any existing one. Using this field we can specify the exact URL to fetch changelogs from.
> Renovate can fetch changelogs from Bitbucket, Gitea (Forgejo), GitHub and GitLab platforms only, and setting the URL to an unsupported host/platform type won't change that.

It seems like the two pages state two different things, or is it two different features altogether?

**Comment by RahulGautamSingh** [comment]
Thanks for catching. I'll fix it. 🫡

**Reply by RahulGautamSingh** [reply]
https://github.com/renovatebot/renovate/pull/30305
